externs/browser/html5.js

```diff
@@ -362,6 +362,17 @@ Path2D.prototype.arcTo = function(x1, y1, x2, y2, radius) {};
  */
 Path2D.prototype.rect = function(x, y, w, h) {};
 
+/**
+ * @param {number} x
+ * @param {number} y
+ * @param {number} w
+ * @param {number} h
+ * @param {(number|DOMPointInit|Array<DOMPointInit>)=} radii
```

Comment:
from mdn and [the spec](https://html.spec.whatwg.org/multipage/canvas.html#dom-context-2d-roundrect), it appears that `radii` can also be an array of numbers, and can mix/match `number`s and `DOMPoint`s?

Also, it doesn't appear that `radii` is optional (i.e. no trailing `=`)? 
```suggestion
 * @param {(number|DOMPointInit|Array<number|DOMPointInit>)} radii
```

Reply:
Agree on adding `number`

My interpretation of the spec is that `radii` is optional though:

![image](https://github.com/google/closure-compiler/assets/749094/bea0760c-d1a3-45cf-9516-f5994107d20b)

```
optional (unrestricted double or [DOMPointInit](https://drafts.fxtf.org/geometry/#dictdef-dompointinit) or sequence<(unrestricted double or [DOMPointInit](https://drafts.fxtf.org/geometry/#dictdef-dompointinit))>) radii = 0
```

Reply:
Ah right you are - in that case both implementations should have it (or just define it once in CanvasPathMethods, reflecting the CanvasPath type in the spec)?